cmd/apichecker/apicheck_test.go

```diff
@@ -0,0 +1,347 @@
+package main
+
+import (
+	"log"
+	"sync"
+	"testing"
+)
+
+func TestCheckExistingExchanges(t *testing.T) {
+	_, _, err := CheckExistingExchanges(jsonFile, "Kraken")
```

Comment:
This will cause tests to also update the updates.json when running tests that will also cause failures if there is nay changes might be best to create "update_test.json" file with static results we can test against